cmd/github-ldap-user-group-creator/main.go

```diff
@@ -290,6 +302,51 @@ type GroupClusters struct {
 
 var githubRobotIds = sets.New[string]("RH-Cachito", "openshift-bot", "openshift-ci-robot", "openshift-merge-robot")
 
+func deleteUsersNotInRover(ctx context.Context, clients map[string]ctrlruntimeclient.Client,
+	kerberosIDs sets.Set[string], dryRun bool) error {
+
+	var errs []error
+	for cluster, client := range clients {
+		usersNotInRover, err := getUsersWithoutKerberosID(ctx, client, cluster, kerberosIDs)
+		if err != nil {
+			errs = append(errs, fmt.Errorf("failed to get users on cluster %s: %w", cluster, err))
+		}
+		for user, hisIdentites := range usersNotInRover {
+			logrus.Infoln("Deleting user", user, "with his identites on cluster", cluster)
```

Comment:
Can we push logging and dryRun logic even further to the loop of identities?
In addition, could we use `logurs.WithField(,)`?